.github/workflows/data-sync-maintainers.yml

```
@@ -11,6 +11,12 @@ on:
     - cron: "0 * * * *"
   workflow_dispatch:
 
+permissions:
+  contents: write
+  pull-requests: write
+  issues: write
+  metadata: read
```

actionlint (1.7.12)

[error] unknown permission scope "metadata". all available permission scopes are "actions", "artifact-metadata", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)